tf_files/modules/cdis-aws-common-logging/lambda_function.py

```diff
@@ -113,17 +110,9 @@ def handler(event, context):
     client = boto3.client('firehose')
```

Comment:
Would you improve the doc-comments on every function and on the file as a while.  In particular - it would be good to know what the format of the input and output to each function is (ex: { "message" : "a comment line", "timestamp": "iso-formated utc timestamp" }), and whether the function modifies the input (like `dateIt` does).

Also - it would be good to know the overall flow - ex 
  - I assume that `handler` is invoked as new message bundles arrive into the Kinesis stream
  - I assume the output of `handler` gets injected into the S3 and Elastic Search fire hoses?

Reply:
The python style guide probably has more tips that I didn't notice :-p :
   https://github.com/uc-cdis/rfcs/blob/master/text/0003-style-guide.md#python-style-guide

Reply:
more documentation in the lambda_function.py itself